yolort/runtime/y_onnxruntime.py

```diff
@@ -49,10 +49,7 @@ def __init__(self, engine_path: str, device: str = "cpu") -> None:
 
     def _set_providers(self):
```

Comment:
that is not important, though we also suggest to move `@_dependency.requires_module("onnxruntime")` before this `self._set_providers()` method.